pkg/format/format.go

```diff
@@ -143,24 +165,31 @@ func findDrives() {
 		// this probably shouldn't be so hard
 		// but d.Mode()&os.ModeDevice == 0 doesn't work as expected
 		mode := d.Sys().(*syscall.Stat_t).Mode
+
 		if (mode & syscall.S_IFMT) != syscall.S_IFBLK {
 			continue
 		}
+
 		// ignore if it matches regexp
 		if ignoreExp.MatchString(d.Name()) {
 			continue
 		}
-		driveKeys = append(driveKeys, filepath.Join("/dev", d.Name()))
-	}
-	sort.Strings(driveKeys)
-	for _, d := range driveKeys {
-		drives[d] = true
+
+		driveKey := filepath.Join("/dev", d.Name())
+
+		// ignore if the deviceVar argument is defined and the driveKey doesn't match the deviceVar
+		if deviceVar != "" && driveKey != deviceVar {
+			continue
+		}
+
+		driveKeys = append(driveKeys, driveKey)
 	}
 }
 
 func init() {
-	flag.StringVar(&labelVar, "label", "", "Disk label to apply")
 	flag.StringVar(&fsTypeVar, "type", "ext4", "Type of filesystem to create")
+	flag.StringVar(&labelVar, "label", "", "Disk label to apply")
+	flag.StringVar(&deviceVar, "device", "", "Device to format")
```

Comment:
new `--device` argument, which is the explicit single device to allow to be formatted

Reply:
nit: these were in alphabetical order before, it would be nice to keep them that way.
`device`, `label`, `type`.